recipes/pasta/1.7.8/build.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+work_dir=$PREFIX/pasta-build
+install_dir=$PREFIX/bin
+
+mkdir -p $work_dir
+mkdir -p $work_dir/pasta
+
+cp -R ./* $work_dir/pasta
+cd $work_dir
+
+unamestr=`uname`
+if [ $unamestr == 'Linux' ];
+then
+    wget --no-check-certificate https://github.com/smirarab/sate-tools-linux/archive/a3e6f56.tar.gz
+    tar -xzf a3e6f56.tar.gz
+    mv sate-tools-linux-a3e6f56372599ffacf1bd43adb2d67cf361228e2 sate-tools-linux
+    cd pasta
+    $PYTHON setup.py install
+    mkdir $install_dir/sate-tools-linux
+    cp -R $work_dir/sate-tools-linux/* $install_dir/sate-tools-linux/
+    # Handle a PASTA bug.
+    cp $install_dir/sate-tools-linux/hmmalign $install_dir/sate-tools-linux/hmmeralign
```

Comment:
@bgruening Believe me, I wish I could have simply depended on these separate recipes.  It would have made figuring out this PASTA recipe much easier!  ;)

This recipe for PASTA installs one of the requirements for this PlantTribes pipeline that I have wrapped in Galaxy: https://github.com/dePamphilis/PlantTribes/blob/master/pipelines/GeneFamilyAligner#L29.

The PASTA installation instructions here https://github.com/smirarab/pasta/blob/master/README.md#2-from-source-code , specifically step 3 which states:
```
Clone the relevant "tools" directory (these are also forked from the SATe project). There are different repositories for linux and MAC. You can use git clone https://github.com/smirarab/sate-tools-linux.git for Linux or git clone https://github.com/smirarab/sate-tools-mac.git for MAC. Or you can directly download these as zip files for Linux or MAC and decompress them in your target directory (e.g. pasta-code)
```
do not allow for installing these SATe tools separately as the PASTA code will not properly compile.